ui/app/AppLayouts/Profile/views/wallet/MainView.qml

```diff
@@ -106,7 +106,6 @@ Column {
         objectName: "generatedAccounts"
         model: walletStore.generatedAccounts
         delegate: WalletAccountDelegate {
-            objectName: "walletSettingsAccountDelegate"
```

Comment:
@jrainville there is already an objectName set inside the walletAccountDelegate file but more importantly, we cannot set a static object name into a list as it prevent us then to find a specifc one. Doing that broke the delete account test.